dist/.eslintrc.json

```diff
@@ -1,9 +1,22 @@
 {
-  "root": true,
+	"root": true,
 
-  "extends": "../.eslintrc-browser.json",
-  "rules": {
-	// That is okay for the built version
-	"no-multiple-empty-lines": "off"
-  }
+	"extends": "../.eslintrc-browser.json",
+	"rules": {
+		// That is okay for the built version
+		"no-multiple-empty-lines": "off",
+
+	  // Sizzle is not compatible with jQuery code style
+	  "no-nested-ternary": "off",
+	  "no-unused-expressions": "off",
+	  "no-unused-vars": "off",
+	  "lines-around-comment": "off",
+	  "space-in-parens": "off",
+	  "camelcase": "off",
+	  "computed-property-spacing": "off",
+	  "max-len": "off",
+	  "dot-notation": "off",
+	  "semi-spacing": "off",
+	  "brace-style": "off"
```

Comment:
This should be left on.

Reply:
Check previous commit about this unnecessary configuration